site/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "es2018",
+    "target": "es2020",
```

Comment:
❓reason for this? I like it but wondering if it was needed or happened in conjunction with Vite addition.

Reply:
Vite only works with es2020+ bundles. Webpack does not break because of that tho.